README.md

```diff
@@ -45,16 +45,19 @@ NPM is the easiest and fastest way to get started using Recharts. It is also the
 
 ```sh
 # latest stable
-$ npm install recharts
+$ npm install recharts react-is
 ```
 
+`react-is` needs to match the version of your installed `react` package.
```

Comment:
peer dependencies are automatically installed, this doesn't need to be in the readme

Reply:
Not by Yarn. This behavior can also be disabled in NPM and PNPM.

Reply:
Hmm...

Hopefully we can just remove the usage of react-is before we release react 19 support. The way its currently used is not ideal/wanted anyways since it supports some ani patterns.

Anyways, appreciate the PR!